sourcecode/apis/contentauthor/app/Http/Middleware/LtiSignedLaunch.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
```

Comment:
Missing `declare`

Reply:
Will work on a more general fix for this later.